cfgov/v1/util/filterable_list.py

```diff
@@ -116,3 +115,9 @@ def form_id(self):
             return form_ids[0]
         else:
             return 0
+
+    def serve(self, request):
```

Comment:
Doesn't this need to be

```py
def serve(self, request, *args, **kwargs):
```

to match the correct signature [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L267) and [here](https://github.com/wagtail/wagtail/blob/master/wagtail/wagtailcore/models.py)?